workflow/rules/common.smk

```diff
@@ -504,25 +476,20 @@ def get_maxins(wildcards):
     if config.bowtie_align["maxins"]:
         return config.bowtie_align["maxins"]
     else:
-        parts = wildcards.dataset.split("/")
-        patient_ID = parts[1]
-        date = parts[2]
+        patient_ID, date = os.path.normpath(wildcards.dataset).split(os.path.sep)[-2:]
         read_len = patient_dict[patient_record(patient_id=patient_ID, date=date)]
         return 4 * read_len
 
 
+# WARNING needs to hand gracefully trailing slashes and re-use the exact two-levels of the directory structure (patient / date)
```

Comment:
"needs to hand gracefully trailing slashes" - does this mean "hand gracefully" as in "gracefully provide"?

Reply:
hand*le* sorry.

"Needs to handle trailing slashes gracefully."